File: python/day_four/perfect_square.py
list =  [4, 9, 25, 49]

def is_perfect_square(list):
	for count in range(0, len(list)):
		number = list[count]
		divisor = 2
		while number % divisor != 0:
			divisor += 1

		dividedIndex = number / divisor

		if number == dividedIndex * dividedIndex:
			list[count] = True
		else:
			list[count] = False
            	
	return list

number =  {4, 9, 25, 49}

print(is_perfect_square(list))
	
		
  

File: java/DayFour/PerfectSquare.java
import java.util.Arrays;

public class PerfectSquare{

 	public static void main(String [] args){
		int[] list =  {4, 9, 25, 49,6};

		System.out.print(Arrays.toString(getPerfectSquare(list)));
	}

	
	public static String[] getPerfectSquare(int [] list) {
        
 
       String[] results = new String[list.length];
	

        for (int count = 0; count < list.length; count++) {
            int number = list[count];
             	int divisor = 2;

                while(number % divisor != 0) {
                    	divisor++;

                }
		int dividedIndex = number / divisor;
		
               	if (number == dividedIndex * dividedIndex) {
                	results[count] = "true";
            	}else{
                	results[count] = "false";
            	}
		}
		
		return results;
	}
	
		
  


      
}











